repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/intrafilterLogging/RequestLog.java

```diff
@@ -41,17 +44,25 @@
     String requestBody;
     HashMap<String, String> headers;
 
+    /**
+     * Constructor populates all of the fields necessary for logging.
```

Comment:
It's not going to keep me up at night, but what value do you see in this? it tells me absolutely nothing that wasn't in already communicated by the names of the parameters